### ISSUE TITLE
[SF] ViewSettingsDialog: A11y focusing issue

### ISSUE THREAD

**Issue by Vir-Mittal** [SAP/ui5-webcomponents-react#2335]
**Describe the bug**

1. In Windows OS, when using NVDA screen reader and using arrow keys to switch between radio button options in the ViewSettingsDialog component, the focus doesn't change, which in MacOS, the focus changes.
2. In MacOS, when using VoiceOver and using arrow keys to switch between radio button options in the ViewSettingsDialog component, the focus changes, but the item name is not read out.

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-viewsettingsdialog--default-story
2. Turn on NVDA
3. Open the dialog
4. Use arrow keys to change the selected options

**Expected behavior**
Using the arrow keys should change which option is focused and that option should be announced clearly on all operating systems.

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: "0.31.21"
`@ui5/webcomponents-react` version: "0.131.23"
`@ui5/webcomponents-fiori` version: "0.31.21"
Operating System: Windows and MacOS
Browser: Chrome

**Additional context**
For SAP Successfactors

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
